### DISCUSSION TITLE
Table View for Media Collection

### DISCUSSION THREAD

**Discussion by zubricks** [payloadcms/payload#1337] [Roadmap]
Two things that would be a nice addition to Media Collection:

1. Table View - this would be especially beneficial for document uploads as they do not have useful thumbnails
2. Toggle for Users to switch between Table & Gallery View (this should be saved to user preferences)

We could possibly achieve the first feature with a new boolean property on `upload.admin`  called `renderListTable` or similar
`renderListTable: true, useListTable: true, list: 'table' | 'thumbnails'`

**Comment by JarrodMFlesch** [comment]
+1 for string values `list: 'table' | 'thumbnails'`
